pydantic_ai_slim/pydantic_ai/models/vertexai.py

```diff
@@ -103,11 +104,16 @@ def __init__(
         self.project_id = project_id
         self.region = region
         self.model_publisher = model_publisher
-        self.http_client = http_client or cached_async_http_client()
+        self.client = http_client or cached_async_http_client()
         self.url_template = url_template
 
         self._auth = None
         self._url = None
+        warnings.warn(
+            'The VertexAIModel is deprecated, please use the GeminiModel with the VertexAIProvider instead.',
```

Comment:
```suggestion
            'The VertexAIModel is deprecated, please use `GeminiModel(provider=VertexAIProvider(...))` instead.',
```
feels like it will help a user who hits this a bit more